### ISSUE TITLE
cp: cannot stat '/usr/share/grub/unicode.pf2'

### ISSUE THREAD

**Issue by oberon2007** [manjaro/manjaro-tools#295]
Seems with updated **grub 2.03.0-6** some adjustment will be needed re **buildiso**:
```
==> Prepare [/iso/boot/grub]
  -> Building core.img ...
  -> Building bootx64.efi ...
cp: cannot stat '/usr/share/grub/unicode.pf2': No such file or directory
```

**Comment by Ste74** [comment]
Yes, all the files .pf2 are not present .. see also the difference in weight of the packages .. first 6 mb now 3 mb
maybe we have to correct **util-iso-boot.sh**

**Comment by Ste74** [comment]
The .pf2 files are used only if we use a graphical theme for grub but in this case ( and this is our case ) we already provide the font to use... if for other reason we need the **unicode.pf2** file we have to do this step in **util-iso-boot.sh** before the cp command we need to install **bdf-unicode** pkg from arch and follow [this](https://wiki.archlinux.org/index.php/GRUB#.22No_suitable_mode_found.22_error) step

**Comment by Ste74** [comment]
Well my assumption 

> The .pf2 files are used only if we use a graphical theme for grub but in this case ( and this is our case ) we already provide the font to use

is wrong .. grub menù 
![grub](https://user-images.githubusercontent.com/11218234/40684523-dc7239ca-6391-11e8-8c73-b65b0ec2318d.png)

so we need to adjust util-iso-boot.sh for do unicode.pf2 or more simply add unicode.pf2 in our grub pkgs ...

**Comment by oberon2007** [comment]
Thanks for all the research !

**Comment by Ste74** [comment]
:stuck_out_tongue_winking_eye: i think the easy way is add the font into the grub folder as source :wink:

**Comment by oberon2007** [comment]
You mean in live-overlay?
I guess both calamares and manjaro-architect will also need adjustment...
Or we need to put it in desktop-overlay? In that case a package will probably be better, because if one day the situation changes and some other pkg decides to deploy the file, we will have conflicts on existing installs ...

**Comment by Ste74** [comment]
No i mean in **grub** and **grub-quiet** in the PKGBUILD

**Comment by philmmanjaro** [comment]
Let me first check what had changed in **grub** and maybe I find a proper solution for it.

**Comment by philmmanjaro** [comment]
Seems some is off:

```
GRUB2 will be compiled with following components:
Platform: x86_64-efi
With devmapper support: Yes
With memory debugging: Yes
With disk cache statistics: Yes
With boot time statistics: Yes
efiemu runtime: No (not available on efi)
grub-mkfont: Yes
grub-mount: Yes
starfield theme: No (No build-time grub-mkfont)
With libzfs support: No (need zfs library)
Build-time grub-mkfont: No (need freetype2 library)
Without unifont (no build-time grub-mkfont)
With liblzma from -llzma (support for XZ-compressed mips images)
```
